client/tailscale/tailscale.go

```diff
@@ -63,6 +65,30 @@ func (c *Client) httpClient() *http.Client {
 	return http.DefaultClient
 }
 
+// BuildURL builds a url to http(s)://<apiserver>/api/v2/... using the given
+// pathElements. It url escapes each path element, so the caller doesn't need
```

Comment:
nit: I had to read the function implementation to make myself clear enough about this comment, maybe this makes it slightly clearer?

```suggestion
// BuildURL builds a url to http(s)://<apiserver>/api/v2/<slash-separated-pathElements>
// It url escapes each path element, so the caller doesn't need
```